src/modules/translation/languages/en.json

```diff
@@ -1039,7 +1039,9 @@
       "hides": "Hides",
       "hides_info": "When a user equips your item, any other items in the category you select will stay equipped, but they won't be rendered. Users will only see their other items reappear after removing your item.",
       "tags": "Tags",
-      "select_placeholder": "Select an option"
+      "select_placeholder": "Select an option",
+      "request_for_changes": "Request for changes",
+      "request_for_changes_explanation": "If you want to make changes to {name} please ask a committee member for a rejection first."
```

Comment:
I would like to avoid showing the internal process which is rejection & approval because may sounds weird. What about
`[...] please ask a committee member for authorization in the forum`?